openfisca_france/model/prestations/minima_sociaux/ass.py

```diff
@@ -10,6 +10,10 @@ class ass_precondition_remplie(Variable):
     entity = Individu
     label = u"Éligible à l'ASS"
     definition_period = MONTH
+    reference = [
+        u"Article R5423-1 du Code du travail",
```

Comment:
In py3, strings are unicode by default.

```suggestion
        "Article R5423-1 du Code du travail",
```